packages/testsuite/cypress/e2e/elytron-oidc-client/test-oidc-security-configure-oidc.cy.ts

```diff
@@ -0,0 +1,158 @@
+describe("TESTS: Configuring Elytron OIDC client in webconsole", () => {
```

Comment:
> Configuring Elytron OIDC client in webconsole

-> Configuration => Subsystem => Elytron OIDC Client

Reply:
Well, maybe not. But I would expect that this spec would verify it is possible/necessary to log in to Web Console using Keycloak then.